app/celery/scheduled_tasks.py

```diff
@@ -383,5 +383,5 @@ def beat_inbox_sms_priority():
 
     while list_of_sms_notifications:
         save_smss.apply_async((None, list_of_sms_notifications, receipt_id_sms), queue=QueueNames.PRIORITY_DATABASE)
-        current_app.logger.info(f"Batch saving with Bulk Priority: SMS receipt {receipt_id_sms} sent to in-flight.")
+        current_app.logger.info(f"Batch saving with Priority: SMS receipt {receipt_id_sms} sent to in-flight.")
```

Comment:
why is this priority? It can be either normal/ bulk or priority?

Reply:
Reading quickly the code, it seems this function is for the SMS high priority notifications? Steve probably removed `Bulk` as this seems a bad copy paste of line 368.

Reply:
yea, exactly, I think we just made beat_inbox_sms_priority() but copying the above beat_inbox_sms_bulk() but forgot the change the log.

I purposely didn't change the bulk / normal / priority in this PR since I figured I'd keep it simple and just add the queues, but 💯 we should change them everywhere to (IMO) low / medium / high